fhir-server-test/src/test/java/com/ibm/fhir/server/test/SystemHistoryTest.java

```diff
@@ -186,10 +187,11 @@ public void testSystemHistoryWithTypePatient() throws Exception {
         int found = 0;
         int count = 10;
         do {
-            Response historyResponse =
-                    target.path(requestPath)
+            Response historyResponse = target.path(requestPath)
                     .queryParam("_count", Integer.toString(count))
-                    .request().get(Response.class);
+                    .request()
+                    .header("X-FHIR-FORCE-UPDATE", "true")
```

Comment:
why add this to a history GET?

Reply:
I added it to a number of tests to force the old behavior...I agree it doesn't make sense for history requests.  Let's remove it.
```suggestion
```

Reply:
agreed